Develop

ValentynaBahyrly opened the pull request:
[DEMO LINK](https://ValentynaBahyrly.github.io/layout_landing-page/)

ValentynaBahyrly committed:
add task solution

ValentynaBahyrly commented:
Не все виконала, є помилки, прошу почекати

Anton-Kuchmasov reviewed (changes requested):
Okay, just re-request a review again, when your solution will be ready for review

ValentynaBahyrly committed:
add task solution

ValentynaBahyrly commented:
> Okay, just re-request a review again, when your solution will be ready for review

Hello. Please check.

etojeDenys reviewed (changes requested):
good job

1. fix phone number position
<img width="124" height="54" alt="image" src="https://github.com/user-attachments/assets/79012611-f7f9-4671-836b-2bd8f06f0144" />

2. add a hover effect to these links according to the layout
<img width="266" height="240" alt="image" src="https://github.com/user-attachments/assets/ea8622d6-b837-4d12-ad99-3cd9aa9796b5" />

3. decrease the scaling slightly to avoid overlap
<img width="293" height="308" alt="image" src="https://github.com/user-attachments/assets/caada847-296b-44e8-9713-3fd301e246d0" />

4. add a cursor pointer to the selectable element

5. make sure that all buttons adhere to the mockup
<img width="471" height="124" alt="image" src="https://github.com/user-attachments/assets/f9aec40e-77f3-4bc9-8069-64ca458673bb" />

6. phone number should only appear on the desktop version
<img width="218" height="227" alt="image" src="https://github.com/user-attachments/assets/32614e4f-8be9-406b-b70b-7b9bfaedd476" />

ValentynaBahyrly committed:
fix phone number position

ValentynaBahyrly committed:
1) fix phone number position
2) add a hover effect to these links according to the layout
3) decrease the scaling slightly to avoid overlap
4) add a cursor pointer to the selectable element
5) make sure that all buttons adhere to the mockup
6) phone number should only appear on the desktop version

ValentynaBahyrly committed:
1) change .form:focus to .form:focus-visible 2) added color $hover-effect-duration in .form:focus-visible

brespect reviewed (changes requested):
Good progress you did, let`s check next:
1. This image should have different positioning according to the mockup:
<img width="1281" height="472" alt="{4F4BE389-4A3A-4060-8316-DBAADC8B53BD}" src="https://github.com/user-attachments/assets/b5dc7ab8-0f0e-4451-b0c8-a3093d4548ac" />

2. Try not to center the placeholder on textarea, let`s add it at the top 
<img width="825" height="516" alt="{ABD4E48E-1170-4448-B9D1-22325716BF41}" src="https://github.com/user-attachments/assets/7fd3ddc1-bb57-44d7-bf02-07c2d8132865" />

3. Provide favicon

ValentynaBahyrly committed:
1) changed scss about: .about-us__photo {background-position},
2) added  scss contact: .form:focus-visible {outline: none},
3)  changed html article-contact-us: <textarea  class="form form--message"  name="message" ... > instead <input type="text">,
4)  changed scss contact: .form {padding: 14px 16px;  font-family: $page-font-family;  font-style: $page-font-style;  font-size: $label-form-font-size;}

brespect reviewed (changes requested):
Not all steps had been passed since previous review:
1. Add the favicon.
2. This pictures onMobile should look different:
<img width="305" height="387" alt="{434DD5B8-1559-4B8C-A8F4-13B623E5A9D1}" src="https://github.com/user-attachments/assets/0635726a-6338-48dd-9366-38cca7e74e2d" />

ValentynaBahyrly committed:
1) Added the favicon,
2) Added .categories__products-forms { border-radius: 4px;}
3) Added .product__photo { object-fit: cover;  object-position: center; border-radius: 4px;}
4) Added .product__photo--categories { height}

etojeDenys reviewed (approved):
great job!
